tinyms/serving/server/server.py

```diff
@@ -78,8 +79,16 @@ class _FlaskServer(object):
     in subprocess.
     """
 
-    def __init__(self):
+    def __init__(self, host='127.0.0.1', port=5000, serving_path=None):
```

Comment:
I think it would be better to remove the `serving_path` process in `_FlaskServer`, instead you could move the related code to `Server` class.

Reply:
You’re right, it’s really something I didn’t think about